tests/test_paperqa.py

```diff
@@ -683,7 +683,7 @@ def test_sparse_embedding(stub_data_dir: Path, vector_store: type[VectorStore])
         citation="WikiMedia Foundation, 2023, Accessed now",
         embedding_model=SparseEmbeddingModel(),
     )
-    assert any(docs.texts[0].embedding)
+    assert any(cast(list, docs.texts[0].embedding))
```

Comment:
Instead of a `cast`, in unit tests it's best to do `assert isinstance(x, list)`, can you do that here?